src/pykoala/data_container.py

```diff
@@ -638,53 +638,25 @@ def rss_variance(self):
     def rss_variance(self, value):
         self.variance = value
 
-    @property
-    def fibre_diameter(self):
-        """Angular diameter of the RSS fibres."""
-        return self._fibre_diameter
-    
-    @fibre_diameter.setter
-    def fibre_diameter(self, value):
-        assert isinstance(value, u.Quantity) or value is None, (
-            "Fibre diameter must be a astropy.units.Quantity")
-        self._fibre_diameter = value
-
-    @property
-    def sky_fibres(self):
-        """Indices of the RSS sky fibres."""
-        return self._sky_fibres
-    
-    @sky_fibres.setter
-    def sky_fibres(self, value):
-        self._sky_fibres = value
-
-    @property
-    def science_fibres(self):
-        """Indices of fibres with a science target."""
-        return np.delete(np.arange(self.intensity.shape[0]), self.sky_fibres)
-
```

Comment:
What is the reason for removing this? This is breaking compatibility with other plotting methods that need to be merged. I am going to pull back this PR for now.